module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -729,6 +730,19 @@ protected function getSimulatedStatus($id, array $patron = null)
 
         if ($json = $this->config['StaticHoldings'][$id] ?? null) {
             foreach (json_decode($json, true) as $i => $status) {
+                if ($status['use_status_class'] ?? false) {
```

Comment:
I wonder if we should document this somewhere -- e.g. Demo.ini above the [StaticHoldings] section, or else in the comments for getRandomHolding, since that's where the ini documentation currently points the user for more information.

Reply:
I added some explanation to the Demo.ini